internal/scripts/ci_build_and_test_freebsd.sh

```diff
@@ -75,19 +75,19 @@ export PATH=$PATH:$HOME/.cargo/bin
 rustup default $RUST_TOOLCHAIN
 # export RUSTFLAGS="-C debug-assertions"
 cargo fmt --all -- --check
-cargo clippy -- -D warnings
+cargo clippy --workspace --exclude iceoryx2-tunnels-zenoh -- -D warnings
```

Comment:
If the `cfg` thing works in the root `Cargo.toml` we could remove this again ... Otherwise we have to adjust also the readme a bit and point out that on freebsd the build process is a bit different and it would be great if we could avoid this.

Reply:
@elfenpiff AFAIK it does not work.

Reply:
@elBoberido Thoughts on this? Should more effort be spent to find another solution (do you happen to know one?)